check/check_db_dump_recent.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+
+"""
+This file contains a script to check the GOCDB failover process is happening.
+
+Specifically, it checks the log file passed as the first arguement for evidence
```

Comment:
Seems there is a typo for arguement -> argument and suceeded -> succeeded.

Reply:
Fixed in [db4a211](https://github.com/GOCDB/gocdb-failover-scripts/pull/22/commits/db4a211a3c4c5ee479bf47236b305dcb85c342af)